spec/integration_spec.rb

```diff
@@ -582,6 +585,24 @@ def self.find(id)
   end
 end
 
+class WorkingEnqueuedObject < ActiveRecord::Base
+  include AlgoliaSearch
+
+  include GlobalID::Identification
```

Comment:
Why do we need to add this line here?

Reply:
ah, yes! i forgot to mention this - copied from the commit message:

```
Specify the GlobalID app name
Required for GlobalID to generate names, but wasn't previously required
as no objects were actually being enqueued and therefore not actually
generating names.

In a rails environment, this would be automatically configured by the
railtie in GlobalID [1].

https://github.com/rails/globalid/blob/b0f5af88e1baf966a4309f97321bf71dcfc6f248/lib/global_id/railtie.rb#L20
```